Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp

```diff
@@ -638,12 +713,16 @@ void ProgramShaderCache::CreateHeader()
           ""
 
       ,
+      !is_glsles && v < GLSL_430 && g_ActiveConfig.backend_info.bSupportsComputeShaders ?
+          "#extension GL_ARB_compute_shader : enable" :
```

Comment:
Oh damn, this fails on nvidia. It expects all shaders as compute shaders now....